src/Evolution/Systems/CurvedScalarWave/Worldtube/SelfForce.hpp

```diff
@@ -105,4 +105,18 @@ tnsr::A<double, Dim> Du_self_force_per_mass(
     const tnsr::A<double, Dim>& dt_self_force,
     const tnsr::A<double, Dim>& four_velocity,
     const tnsr::Abb<double, Dim>& christoffel);
+/*!
+ * \brief Computes the time derivative of the covariant derivative of the scalar
+ * self-force per unit mass $f^\alpha along$ the four velocity $u^\beta$, i.e.
+ * $\frac{d}{dt}u^\beta \nabla_\beta f^\alpha$.
```

Comment:
Reference `Du_self_force_per_mass` here as well. Also, you have the word "along" inside the tex. Move that outside please.